src/styles/blocks/about-us.scss

```diff
@@ -0,0 +1,69 @@
+.about-us-image {
+  height: 200px;
+  width: 100%;
+  object-fit: cover;
+  object-position: center -90px;
+
+
+  display: block;
+
+  @media (min-width: 744px) {
```

Comment:
try to use mixins for all media query

Reply:
i will but its too long to rework all that ive done